contrib/create-basic-configdrive

```diff
@@ -55,6 +55,13 @@ if [ $(id -u) -eq 0 ]; then
     exit 1
 fi
 
+# mkisofs tool check
+which mkisofs &>/dev/null
+if [ $? -ne 0 ]; then
```

Comment:
Ok, I'll change it. I tried to be consistent, you have this style in `create-coreos-vdi` script when checking for VirtualBox tools.